src/templates/script/getScriptVerifiedTemplateIds.ts

```diff
@@ -26,6 +26,8 @@ export function getScriptVerifiedTemplateIds(version: string): (string | RegExp)
         verifiedTemplateIds = verifiedTemplateIds.concat([
             'CosmosDBTrigger',
             'DurableFunctionsActivity',
+            'DurableFunctionsEntity',
+            'DurableFunctionsEntityHttpStart',
```

Comment:
We would have to remove this from the verified template id for Python specifically until we don't skip the test.

Reply:
@nturinski: does this latest commit [here](https://github.com/microsoft/vscode-azurefunctions/pull/3412/commits/12306666105bdc23598681b420cb51a3c26b9cf2) match what you're looking for?